Shared.Tests/ApiTest.cs

```diff
@@ -1490,5 +1491,53 @@ public void TestSignParameters()
 
             StringAssert.AreNotEqualIgnoringCase(m_api.SignParameters(paramsSetOne), m_api.SignParameters(paramsSetTwo), "The signatures are equal.");
         }
+
+        [Test]
+        public void TestVerifyApiResponseSignature()
+        {
+            const string publicId = "resource id";
```

Comment:
We prefer to use the same test input values across SDKs to make sure we get the same output (if possible)